src/App.js

```diff
@@ -5,16 +5,48 @@ import Button from "@mui/material/Button";
 import CssBaseline from "@mui/material/CssBaseline";
 import Box from "@mui/material/Box";
 import Container from "@mui/material/Container";
-import TodoBox from "./components/TodoBox";
+import TodoForm from "./components/TodoForm";
+import Typography from "@mui/material/Typography";
+import Paper from "@mui/material/Paper";
+import AppBar from "@mui/material/AppBar";
+import Toolbar from "@mui/material/Toolbar";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+// import CheckboxList from "./components/CheckboxList";
```

Comment:
It is a good practice remove this kind of comments

Reply:
done